doc/VMSpecificationSingleNode

```diff
@@ -0,0 +1,15 @@
+### VM Specifications - Single-Node VM deployment
+
+- Please create VM(s) with at least 1 Node, 2 cores (vCPU) and 4GB of RAM.
+- Ensure the VM is created with minimum 6 attached disks (raw) per node i.e. /dev/sdb (Data Disk), /dev/sdc (Metadata Disks)
+
+  - Data Disks - Min 4 Disks per node (Capacity 10G+)
+  - Metadata Disks - Min 2 Disks per node (Capacity - 10% of total Data Disk Size)
+  
+- Ensure the VM is created with minimum 3 network Interfaces (recommended 5 interfaces)
+
+  - private interfaces: ens36,ens37 
```

Comment:
The names won't be the same always.
Plus, we don't need 2 but just one interface value per key here (since we don't expect teaming on OVA).

Reply:
done